docs/user/whois.rst

```diff
@@ -30,12 +30,15 @@ associated with the device's public IP address and includes:
 - Timezone of the ASN's registered location
 - Coordinates (Latitude and Longitude)
 
+.. _whois_trigger_conditions:
```

Comment:
```suggestion
.. _controller_whois_trigger_conditions:
```

While the possibility of conflicts is low for this label, prefixing with _controller helps to identity that something is present in controller (when you're working on another module).